package.json

```diff
@@ -56,7 +56,7 @@
     "husky": "4.2.4",
     "jest": "24.9.0",
     "mock-stdin": "1.0.0",
-    "nock": "12.0.3",
+    "nock": "13.0.4",
```

Comment:
I updated nock during the investigations, it works well with this version, we should keep it.
I know this is something that should ideally be isolated in a dedicated contribution.